src/main/java/com/aws/greengrass/status/FleetStatusService.java

```diff
@@ -295,24 +296,40 @@ public void schedulePeriodicFleetStatusDataUpdate(boolean isDuringConnectionResu
                 initialDelay, periodicPublishIntervalSec, TimeUnit.SECONDS);
     }
 
+    private AtomicReference<OverallStatus> getOverAllStatus() {
```

Comment:
i think `overall` is just one word so your method should be named `getOverallStatus`

Reply:
Updated